modules/windows-workload-management.adoc

```diff
@@ -24,4 +24,11 @@ image::mixed-windows-linux-workloads.png[Mixed Windows and Linux workloads]
 
 The WMCO expects a predetermined secret in its namespace containing a private key that is used to interact with the Windows instance. WMCO checks for this secret during boot up time and creates a user data secret which you must reference in the Windows `MachineSet` object that you created. Then the WMCO populates the user data secret with a public key that corresponds to the private key. With this data in place, the cluster can connect to the Windows VM using an SSH connection.
 
-Once the cluster establishes a connection with the Windows VM, you can manage the Windows node using similar practices as you would a Linux-based node. Scheduling Windows workloads to a Windows node can be done with typical pod scheduling practices like taints, tolerations, and node selectors, but you must also differentiate your Windows workloads from Linux workloads and other Windows-versioned workloads by using a `RuntimeClass` object.
+Once the cluster establishes a connection with the Windows VM, you can manage the Windows node using similar practices as you would a Linux-based node.
```

Comment:
s/Once/After/ 

per IBMSG

Reply:
Not sure whether I think it's warranted, but maybe spell out "virtual machine (VM)" for wherever the first instance in this module is.